src/platform/user/profile/vet360/components/EmailField/EmailField.jsx

```diff
@@ -6,16 +6,43 @@ import { API_ROUTES, FIELD_NAMES } from 'vet360/constants';
 import { isValidEmail } from 'platform/forms/validations';
 
 import Vet360ProfileField from 'vet360/containers/Vet360ProfileField';
-import EmailEditModal from './EmailEditModal';
 
+import EmailEditModal from './EmailEditModal';
 import EmailView from './EmailView';
 
+const formSchema = {
+  type: 'object',
+  properties: {
+    emailAddress: {
+      type: 'string',
+      // This regex was taken from the HCA but modified to allow leading and
+      // trailing whitespace to reduce false errors. The `convertDataToPayload`
+      // method will clean up the whitespace before submission
+      pattern:
+        '^(\\s)*(([^<>()\\[\\]\\\\.,;:\\s@"]+(\\.[^<>()\\[\\]\\\\.,;:\\s@"]+)*)|(".+"))@((\\[[0-9]{1,3}\\.[0-9]{1,3}\\.[0-9]{1,3}\\.[0-9]{1,3}\\])|(([a-zA-Z\\-0-9]+\\.)+[a-zA-Z]{2,}))(\\s)*$',
```

Comment:
So this is better than `email.indexOf('@') > 0`?

Reply:
Well it checks for the .tld part as well. It seems like a solid regex in my testing

Reply:
Completely kidding, the regex is way better I'm sure 🙂 My code would would be just fine with `_@` as an email address lol

Reply:
Wow, I must be in a weird headspace to have completely missed the sarcasm 😬